From 03eb84185d1437186c1232011cad871bffa3611e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20M=C3=ADchal?= <harrymichal@seznam.cz>
Date: Sat, 6 Sep 2025 22:27:47 +0300
Subject: [PATCH] Set CLI command for 'git webhook create' to lower-case

Seems like the original casing was a mistake, so let's fix that.
---
 git.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/git.go b/git.go
index 44db8a5..dd9bb4b 100644
--- a/git.go
+++ b/git.go
@@ -1063,7 +1063,7 @@ func newGitWebhookCreateCommand() *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:               "Create [repo]",
+		Use:               "create [repo]",
 		Short:             "Create a git webhook",
 		Args:              cobra.MaximumNArgs(1),
 		ValidArgsFunction: completeGitRepo,
-- 
2.51.0

